sdk/core/Azure.Core/api/Azure.Core.net5.0.cs

```diff
@@ -516,6 +516,17 @@ public partial class BearerTokenAuthenticationPolicy : Azure.Core.Pipeline.HttpP
         public override void Process(Azure.Core.HttpMessage message, System.ReadOnlyMemory<Azure.Core.Pipeline.HttpPipelinePolicy> pipeline) { }
         public override System.Threading.Tasks.ValueTask ProcessAsync(Azure.Core.HttpMessage message, System.ReadOnlyMemory<Azure.Core.Pipeline.HttpPipelinePolicy> pipeline) { throw null; }
     }
+    public partial class BearerTokenChallengeAuthenticationPolicy : Azure.Core.Pipeline.HttpPipelinePolicy
+    {
+        public BearerTokenChallengeAuthenticationPolicy(Azure.Core.TokenCredential credential, System.Collections.Generic.IEnumerable<string> scopes) { }
+        public BearerTokenChallengeAuthenticationPolicy(Azure.Core.TokenCredential credential, string scope) { }
+        protected string[] Scopes { get { throw null; } }
```

Comment:
We shouldn't expose a mutable array.

Reply:
Changed - lemme know what you think.

Reply:
Do we really need it? Can the inheritor grab the `scopes` list from the ctor parameters on their own?

Reply:
They can change after construction.